### ISSUE TITLE
Nodejs - ReferenceError: window is not defined

### ISSUE THREAD

**Issue by ion-willo** [RasCarlito/axios-cache-adapter#33]
Unable to use this adaptor on my server as I get:

```
node_modules/axios-cache-adapter/dist/cache.js:10})(window, function(__WEBPACK_EXTERNAL_MODULE_axios__, __WEBPACK_EXTERNAL_MODULE_lodash_omit__, __WEBPACK_EXTERNAL_MODULE_lodash_merge__, __WEBPACK_EXTERNAL_MODULE_lodash_isFunction__, __WEBPACK_EXTERNAL_MODULE_lodash_isString__, __WEBPACK_EXTERNAL_MODULE_lodash_size__, __WEBPACK_EXTERNAL_MODULE_lodash_map__, __WEBPACK_EXTERNAL_MODULE_lodash_extend__, __WEBPACK_EXTERNAL_MODULE_lodash_find__, __WEBPACK_EXTERNAL_MODULE_lodash_isEmpty__) {
   ^

ReferenceError: window is not defined
    at Object.<anonymous> (node_modules/axios-cache-adapter/dist/cache.js:10:4)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.axios-cache-adapter (build/server.js:15778:18)
    at __webpack_require__ (build/server.js:630:30)
```

**Comment by RasCarlito** [comment]
Hi @ion-willo yes I noticed this 😞 

I am currently looking for a fix. Definitely something with the webpack configuration.

For now you may revert to the `2.0.0` tag which does not have this problem to my knowledge.

If you'd like to help finding a solution you are very welcome!

Cheers

**Comment by RasCarlito** [comment]
The [`target: 'web:` in the webpack config](https://github.com/RasCarlito/axios-cache-adapter/blob/master/webpack.config.js#L90) seems to be the culprit for this bug.

But as stated in the [webpack targets doc](https://webpack.js.org/concepts/targets/#multiple-targets), webpack does not support multiple targets.

Having a different build for each target does not sound great to me, but maybe there is no other way.

If it is the only way then the steps for fixing this should be:

* Creating a specific webpack config for each target: web and node
* Have it build to `dist/cache.js` and `dist/cache.node.js` respectively
* Add a `browser: 'dist/cache.js'` field in package.json
* Change package.json `main` entry to point to `dist/cache.node.js`

Help on this would be greatly appreciated 😄

**Comment by ion-willo** [comment]
Thanks for the update @RasCarlito
Sorry I couldn't be of more help - project pressures and all that.

**Comment by RasCarlito** [comment]
Will release a new version with the fix ASAP 😃

**Comment by RasCarlito** [comment]
@ion-willo Just released [v2.1.1](https://github.com/RasCarlito/axios-cache-adapter/releases/tag/2.1.1) with the node fix
